eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -98,10 +98,15 @@ function Get-GitHubPullRequest {
     $PullRequestNumber,
     [ValidateNotNullOrEmpty()]
     [Parameter(Mandatory = $true)]
-    $AuthToken
+    $AuthToken,
+    [Boolean]$ReturnFiles=$false #Pass true to return the files in the PR
   )
 
   $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/pulls/$PullRequestNumber"
+  if ($ReturnFiles) 
+  {
+    $uri += "/files"
```

Comment:
This change looks unrelated to the other change. What is the purpose of this change?

Reply:
Was going to use it then figured I did not need to. I still think its a good feature though.

Reply:
Lets remove it until we have some that needs to use it.